tests/tox.sh

```diff
@@ -93,16 +93,15 @@ if [[ "$CEPH_STABLE_RELEASE" != "$CURRENT_CEPH_STABLE_RELEASE" ]]; then
 fi
 
 echo "Building flavor $FLAVOR"
-sudo make FLAVORS="$FLAVOR" build.parallel
-
-current_branch="$(git rev-parse --abbrev-ref HEAD)"
-# build tag from luminous,centos,7 to luminous-centos-7-x86_64
-TAG="${current_branch}-$(echo "$FLAVOR"|awk -F ',' '{ print $1"-"$3"-"$4"-"$2}')"
+make_output=$(sudo make FLAVORS="$FLAVOR" build.parallel)
+daemon_image=$(echo "${make_output}" | grep "DAEMON_IMAGE ") # Find DAEMON_IMAGE line
+daemon_image="${daemon_image#*DAEMON_IMAGE*: }" # Remove DAEMON_IMAGE from beginning
+daemon_image="$(echo "${daemon_image}" | tr -s ' ')" # Remove whitespace
 
 # start a local docker registry
 sudo docker run -d -p 5000:5000 --restart=always --name registry registry:2
 # add the image we just built to the registry
-sudo docker tag ceph/daemon:"$TAG" localhost:5000/ceph/daemon:"$CEPH_STABLE_RELEASE"-latest
+sudo docker tag ${daemon_image} localhost:5000/ceph/daemon:"$CEPH_STABLE_RELEASE"-latest
```

Comment:
Sloppy reports an error here :+1: 
In /tox.sh line 104:
sudo docker tag ${daemon_image} localhost:5000/ceph/daemon:"$CEPH_STABLE_RELEASE"-latest
                ^-- SC2086: Double quote to prevent globbing and word splitting.

Reply:
Fixed.